### PR TITLE
assignment evaluates from left to right

### PR DESCRIPTION
**Pull request by dneto0**
- update "simple assignment" description
- update uniformity analysis

Fixes: #3928

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -6656,23 +6656,36 @@ first a [=read access=] gets the old value, and then a [=write access=] stores t
   </xmp>
 </div>
 
-Note: A compound assignment can rewritten as different WGSL code that uses a [=simple assignment=] instead.
+<div class=note><span class=marker>Note:</span>
```
Comment (dneto0) [inline wgsl/index.bs:6659]: This part is a drive-by editorial polish.

Make it one div so the three paragraphs are seen as one big note.
And break the code fragments out to their own blockquotes.

File: wgsl/index.bs [hunk 2]
```diff
@@ -10383,14 +10396,14 @@ When several edges have to be created we use `X -> {Y, Z}` as a short-hand for `
 
       For each [=address spaces/function=] address space pointer parameter *i*,
       [=Value_return_i_contents=] -> *Vin*(*prev*) (see [[#func-var-value-analysis]])
-  <tr><td class="nowrap">*e2* = *e1*;
+  <tr><td class="nowrap">*e1* = *e2*;
```
Comment (dneto0) [inline wgsl/index.bs:10399]: This matches https://dawn-review.googlesource.com/c/dawn/+/123120/4/src/tint/resolver/uniformity.cc#553 
which was independently developed.

**Review by jrprice — APPROVED** [review]
LGTM

**Review by alan-baker — APPROVED** [review]
